intranet/apps/eighth/management/commands/dev_create_blocks.py

```diff
@@ -0,0 +1,59 @@
+from datetime import date, timedelta
+
+from django.core.management.base import BaseCommand, CommandError
+
+from intranet.apps.eighth.models import EighthActivity, EighthBlock, EighthScheduledActivity
+
+
+class Command(BaseCommand):
+    help = "Generate 8th Period Blocks"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("end_date")
+
+    def handle(self, *args, **kwargs):
+        try:
+            end_month, end_day, end_year = kwargs["end_date"].split("/")
+        except OSError as ex:
+            raise CommandError(str(ex)) from ex
+
+        wed_a_activities = EighthActivity.objects.filter(wed_a=True)
+        wed_b_activities = EighthActivity.objects.filter(wed_b=True)
+        fri_a_activities = EighthActivity.objects.filter(fri_a=True)
+        fri_b_activities = EighthActivity.objects.filter(fri_b=True)
+
+        curr_date = date.today()
+        end_date = date(int(end_year), int(end_month), int(end_day))
+        delta = timedelta(days=1)
+
+        while curr_date <= end_date:
```

Comment:
From this code, it appears that the behavior you are trying to achieve is using the `wed_a`, `web_b`, .. fields on `EighthActivity`s in order to create the appropriate `EighthBlock`s and `EighthScheduledActivity`s. Could you document this behavior in the help text and elsewhere in this script?